### PR TITLE
Footer updates

### PR DESCRIPTION
**Pull request by vazquez**

### CODE REVIEW

File: static/img/twitter.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (thisandagain) [inline static/img/twitter.svg]: Mind running this through SVGO?

**Comment by thisandagain** [comment]
One small optimization note and then R+

File: static/img/twitter.svg (hunk 1 above)
Comment (vazquez) [inline static/img/twitter.svg]: Definitely :+1:

**Comment by vazquez** [comment]
Done! Will merge once Travis passes

**Pull request merged**